plot_scheduler_results.py

```
@@ -0,0 +1,234 @@
+#!/usr/bin/env python3
+"""
+Stride Scheduler Results Plotter
+Generates graphs showing CPU distribution over time for stride scheduling test
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import sys
+import re
```

Ruff (0.12.2)

`re` imported but unused

Remove unused import: `re`

(F401)